packages/client/hmi-client/src/components/workflow/ops/interventions/tera-intervention-card.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+	<div class="intervention-card">
+		<header class="flex align-items-center">
+			<tera-edit-value
+				class="mr-auto"
+				:model-value="intervention.name"
+				@update:model-value="onUpdateName($event)"
+				tag="h5"
+			/>
+			<RadioButton
+				:model-value="interventionType"
+				:input-id="uniqueId()"
+				name="interventionType"
+				value="static"
+				@click="onInterventionTypeChange('static')"
+			/>
+			<label for="static" class="ml-2">Static</label>
+			<RadioButton
+				:model-value="interventionType"
+				:input-id="uniqueId()"
+				name="interventionType"
+				value="dynamic"
+				class="ml-3"
+				@click="onInterventionTypeChange('dynamic')"
+			/>
+			<label for="dynamic" class="ml-2">Dynamic</label>
+			<Button class="ml-3" icon="pi pi-trash" text @click="emit('delete')" />
+		</header>
+		<section>
+			<!-- Static -->
+			<template v-if="interventionType === 'static'">
+				<div class="flex align-items-center flex-wrap gap-2">
+					<span>Set Parameter</span>
+					<Dropdown
+						:model-value="intervention.appliedTo"
+						@change="onAppliedToParameterChange"
+						:options="parameterOptions"
+						option-label="label"
+						option-value="value"
+					/>
+					<span>to<span v-if="intervention.staticInterventions.length > 1">...</span></span>
+					<template v-if="intervention.staticInterventions.length === 1">
```

Comment:
Why does `interventions.length` matter?

Reply:
Apparently when length==1 the design sketch has a slightly different style ... will leave this with @blanchco to do what is appropriate.